bigbone/src/main/kotlin/social/bigbone/api/method/StatusMethods.kt

```diff
@@ -44,15 +45,19 @@ class StatusMethods(private val client: MastodonClient) {
     }
 
     /**
-     * Fetch preview card.
-     * @param statusId ID of the status.
-     * @see <a href="https://docs.joinmastodon.org/methods/statuses/#card">Mastodon API documentation: methods/statuses/#card</a>
+     * Translate the status content into some language.
+     * @param statusId The ID of the Status in the database.
+     * @param language ISO 639 language code. The status content will be translated into this language. Defaults to the user’s current locale.
+     * @see <a href="https://docs.joinmastodon.org/methods/statuses/#translate">Mastodon API documentation: methods/statuses/#translate</a>
      */
     @Throws(BigBoneRequestException::class)
-    fun getCard(statusId: String): MastodonRequest<PreviewCard> {
+    fun translateStatus(statusId: String, language: String? = null): MastodonRequest<Translation> {
```

Comment:
JvmOverloads annotation should be added here.

Reply:
Regarding the `language` parameter: Does it make sense to call this method, but have the `language` parameter optional? Shouldn't we make this mandatory?

Reply:
Mastodon documentation states that this "defaults to the user’s current locale" - which, I assume, means the "interface language" defined in the user's Mastodon preferences. If that is the case, I think it makes sense to have a null default here so that not every call to translate would need to get the user's locale first.

Documentation doesn't state much more than that, though, so I think we'll have to find out if the assumption is correct. ;)

Reply:
> Documentation doesn't state much more than that, though, so I think we'll have to find out if the assumption is correct. ;)

Ah, I see. Sure, then it's fine of course. 🙂